### PR TITLE
add option to place signature above quoted text

### PR DESCRIPTION
**Pull request by zimmersi**
Signed-off-by: Silvio Zimmer <silvio.zimmer@die-kinderwelt.com>

### CODE REVIEW

File: src/components/SignatureSettings.vue [hunk 1]
```diff
@@ -93,6 +104,27 @@ export default {
...
+	watch: {
+		async signatureAboveQuote(val, oldVal) {
+			await this.$store
+				.dispatch('patchAccount', {
+					account: this.account,
+					data: {
+						signatureAboveQuote: val,
+					},
+				})
+				.then(() => {
```
Comment (ChristophWurst) [inline src/components/SignatureSettings.vue:116]: watch out, you are mixing async/await with promises.

if you await the dispatch then wrap it with a try-catch for the error handling. you can then use the code as if it was synchronous

Comment (zimmersi) [reply src/components/SignatureSettings.vue:116]: omg, thx for pointing this out

**Review by kesselb — APPROVED** [review]

**Comment by kesselb** [comment]
Thanks :+1:

**Comment by zimmersi** [comment]
Thanks! 👍

**Pull request merged**